Modules/Synchronization/src/Synchronization.Domain/DomainEvents/Incoming/RelationshipReactivationCompleted/RelationshipReactivationCompletedDomainEvent.cs

```diff
@@ -3,7 +3,7 @@
 namespace Backbone.Modules.Synchronization.Domain.DomainEvents.Incoming.RelationshipReactivationCompleted;
 public class RelationshipReactivationCompletedDomainEvent : DomainEvent
 {
-    public RelationshipReactivationCompletedDomainEvent(string relationshipId, string peer) : base($"{relationshipId}/Reactivation/Completed")
+    public RelationshipReactivationCompletedDomainEvent(string relationshipId, string peer) : base($"{relationshipId}/ReactivationCompleted")
```

Comment:
No need for the id in case of incoming events

```suggestion
    public RelationshipReactivationCompletedDomainEvent(string relationshipId, string peer)
```